deploy/olm-catalog/3scale-operator/3scale-operatorsource.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: three-scale-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: 3scale
```

Comment:
You might want to create a new quay org for this purpose. Given that you use the 3scale org in quay to store images, you can't have an application repo of the same name.